examples/vision/bit.py

```diff
@@ -0,0 +1,255 @@
+"""
+Title: Image Classification using BigTransfer(BiT)
+Author: [Sayan Nath](https://twitter.com/sayannath2350)
+Date created: 2021/09/22
+Last modified: 2021/09/22
+Description: BigTransfer (BiT) State-of-the-art transfer learning for image classification.
+"""
+
+"""
+## Introduction
+"""
+
+"""
+BigTransfer known as BiT is a State-of-the-art transfer learning method for Image Classification Tasks. 
+A set of pre-trained image models that can be transferred to obtain excellent performance on new datasets, 
```

Comment:
"A set of pre-trained image models that can be transferred to obtain excellent performance on new datasets" -> please elaborate on what makes BiT different from other transfer learning methods.